.husky/pre-commit

```diff
@@ -5,4 +5,4 @@ npm run check-secrets
 
 npm run lint-staged
 
-npm run full-hbs-lint
+# npm run full-hbs-lint
```

Comment:
I like that (I don't think we should run lint in the whole codebase of a package), but what's the context for this change? To avoid unnecessary processing upon commit? Did we try `lint-staged` for that?

Reply:
Also was removed after. The linter (I added that full-hbs-lint to help us catch handlebars syntax errors) has been throwing errors for me unnessecarily and its been on my to do list to fix it but I havent gotten around to it, so I just have been commenting it otherwise stops me from committing

Reply:
Gotcha.

Just tested it again, this is pretty quick, so not an issue to run on each commit:

![image](https://github.com/metriport/metriport/assets/2132564/20c0ac85-5e41-4a60-bfe2-ecc15166b76b)